lib/state_of_the_nation.rb

```diff
@@ -30,6 +30,14 @@ def until(finish_key)
           (finish.blank? || round_if_should(finish) > round_if_should(time)) && round_if_should(start) <= round_if_should(time)
         end
 
+        define_method "active_in_interval?" do |interval_start:, interval_end: nil|
+          start_before_interval_finish_or_comparison_with_nil = (start.nil? || interval_end.nil? || round_if_should(start) < interval_end)
+          finish_after_interval_start_or_comparison_with_nil = (finish.nil? || interval_start.nil? || round_if_should(finish) > interval_start)
+          start_and_finish_not_equal_or_are_nil = (start != finish || start.nil? || finish.nil?)
+          return true if start_before_interval_finish_or_comparison_with_nil && finish_after_interval_start_or_comparison_with_nil && start_and_finish_not_equal_or_are_nil
+          return true if start_before_interval_finish_or_comparison_with_nil && finish_after_interval_start_or_comparison_with_nil && !ignore_empty
```

Comment:
I think this should return an explicit `false` at the end? Because right now it returns `nil` which is `falsey` but not `false` :D

Reply:
Good shout!